README Changes

tomlazar opened the pull request:
Added install directions to the readme.

ErikBoesen commented:
Okay, please rebase into one [well-titled](https://github.com/frc1418/guidelines/blob/master/github.md#commiting) commit and you're good to go.

Commit:
add install directions

made changes to readme according to feedback

added install directions to the readme
